### DISCUSSION TITLE
Notify existing users when they're added to a team

### DISCUSSION THREAD

**Discussion by rimi-itk** [pretix/pretix#2011] [Ideas]
On [https://docs.pretix.eu/en/latest/user/organizers/teams.html](https://docs.pretix.eu/en/latest/user/organizers/teams.html#user-teams:~:text=If%20the%20user%20already%20has%20an,as%20many%20teams%20as%20you%20want) is says implicitly that existing users do not get any notification when they're added to a team:

> If the user (added to the team, Ed.) already has an account in the pretix system they will instantly get access to the team. Otherwise, they will be sent an email with an invitation link that can be used to create an account. This account will then instantly have access to the team. 

Our users (and administrators) have been a little confused about this (lack of notification), and we therefore suggest that

*existing users should get an email notification when they're added to a team.*

Maybe a setting should control whether or not a notification is sent to existing users. Or maybe a ”Notify user about this team” button next to a member on the team member list could trigger the notification.

Best regards,
Mikkel

**Comment by raphaelm** [comment]
I don't even think we should add a button or setting, I think we should just send an email in that case.

**Reply by rimi-itk** [reply]
Always sending an email will definitely work for us.

**Reply by akolaarthurali** [reply]
I would prefer a pop-up notification or an alert in the  notifications section

**Reply by raphaelm** [reply]
Which notifications section? So far pretix does not have one
